### ISSUE TITLE
Feat: Multimoose

### ISSUE THREAD

**Issue by LalithShiyam** [QIMP-Team/MOOSE#5]
Currently MOOSE is running on server configuration. So there is a good chance that the user is using a DGX or so. In that case, it would make sense to fully utilise the capabilities of the hardware. Similar to falcon, moose should run in parallel based on the hardware capabilities.

**Comment by Keyn34** [comment]
Nope, this is only convenience, I think. MOOSE is, by default, capable of running in parallel when started in multiple terminals on multiple datasets, right? So multimoose is, in theory, a "wrapper" for that.

Would you like to have it? I would say that's a very low priority still.

**Comment by LalithShiyam** [comment]
Nope, not really. Let’s close this. We will not work on this :)
